src/rpc/blockchain.cpp

```diff
@@ -1413,10 +1413,76 @@ UniValue reconsiderblock(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+UniValue getchaintxstats(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() > 2)
+        throw runtime_error(
```

Comment:
looks like travis complains now: the `using namespace` was removed, so this should be `std::runtime_error`, will fix this up before merging